networks.json

```diff
@@ -31,6 +31,13 @@
     "type": "testnet",
     "network": "eos",
     "endpoints": [
+      {
+        "name": "Cryptolions.io",
+        "protocol": "http",
```

Comment:
We don't support HTTP endpoints from the toolkit

Reply:
Ok. I give up.